src/main/webapp/js/statusMessage.js

```diff
@@ -1,3 +1,14 @@
 $(document).ready(function() {
-	document.getElementById('statusMessage').scrollIntoView();
+    var statusMessage = document.getElementById('statusMessage');
+    var navbarHeight = '70';
```

Comment:
this shouldn't be a string. It will also be better if this height was obtained programmatically.